### ISSUE TITLE
Version issue

### ISSUE THREAD

**Issue by zslucky** [Microsoft/react-native-code-push#727]
Steps:
1. Run `npm install --save react-native-code-push@latest`
2. Add `pod 'CodePush', :path => '../node_modules/react-native-code-push'` to Podfile
3. Run `pod install`

Got:
`[!] Invalid `CodePush.podspec` file: Malformed version number string v1.17.1.`

See code in `CodePush.podspec`:
`package['version'].sub('-beta', '')` will return `v1.17.1`.

I manually use 1.17.1 instead of it, then it can work, but I'm not sure is it correct for others? I'm not familiar with CocoaPod. Looks like a bug.

**Comment by sergey-akhalkov** [comment]
@zslucky, I've tried to reproduce the issue using provided repro steps but with no luck. 
As for me I have following after executing `react-native init myapp` and `npm install --save react-native-code-push@latest` commands:
```json
"dependencies": {
    "react": "15.4.2",
    "react-native": "0.41.2",
    "react-native-code-push": "^1.17.1-beta"
  },
```
Could you please look at your version of `react-native-code-push` in `package.json` file in `dependencies` list? It should be the same as for me - `^1.17.1-beta`

**Comment by zslucky** [comment]
Hi @sergey-akhalkov , thanks a lot, I got the same dependency.
```json
"dependencies": {
    "react": "15.4.2",
    "react-native": "0.41.2",
    "react-native-code-push": "^1.17.1-beta"
  },
```

but I think it's a CocoaPods issue, when I run the `pod install`, the `CodePush.podspec` file in your repo using `s.summary = package['version'].sub('-beta', '')` and the package.json in your repo is using version `v1.17.1-beta`, so I think this is the issue.

**Comment by sergey-akhalkov** [comment]
@zslucky, oh, I see, looks like the problem is in leading `v` letter: **v**1.17.1-beta. I've found the same issue here: https://github.com/CocoaPods/CocoaPods/issues/1687
I also could confirm that previous version of `react-native-code-push` does not use leading `v` letter in the [version prop](https://github.com/Microsoft/react-native-code-push/blob/v1.17.0-beta/package.json#L3). And current version (`v1.17.1-beta`) [use it](https://github.com/Microsoft/react-native-code-push/blob/v1.17.1-beta/package.json#L3).
But I still haven't reproduced the issue - my `../node_modules/react-native-code-push/package.json` file contains well formatted version prop: `"version": "1.17.1-beta"` (I've installed `react-native-code-push` package using `npm install --save react-native-code-push@latest` command).
By the way, I think It would be great to fix this issue, because for now it is not possible to use local copy or remote repo of `react-native-code-push` for installation using podFile.

Thank you so mush!
